05.md

```
@@ -0,0 +1,580 @@
+
+**Example - APNs token (32 bytes)**:
+```
+platform_byte  = 0x01
+token_length   = 0x00 0x20  (32 in big-endian)
+device_token   = <32 bytes>
+random_padding = <185 random bytes>
+Total: 1 + 2 + 32 + 185 = 220 bytes
+```
+
+**Example - FCM token (200 bytes)**:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)